doc/documentation/installation.rst

```diff
@@ -157,9 +137,22 @@ This script installs `pre-commit <https://pre-commit.com/>`_ and sets up the pre
 Configure
 ~~~~~~~~~
 
-|FOURC| enforces an out-of-source build, i.e. your build directory may not be located inside the source code directory.
+|FOURC| enforces an out-of-source build, i.e. your build directory may not be the same as your source directory.
+This is a good practice to keep your source directory clean. Instead, create a build directory.
+Many development tools work well when the build directory is a subdirectory of the source directory.
+A possible directory structure could look like this::
+
+::
+
+    <4C_home>/
+       build/
+         debug/
+         release/
+         other_configuration
+
+That said, you can create a build directory wherever you want. This is just a suggestion we find useful in daily work.
 
-|FOURC| uses ``cmake`` with appropriate presets for the build process.
+|FOURC| uses ``cmake`` for configuration and creation of a build. We strongly recommend to use preset files for ``cmake``.
```

Comment:
Maybe add a short comment where you can find preset files. Maybe something like:
```suggestion
|FOURC| uses ``cmake`` for configuration and creation of a build. We strongly recommend to use preset files for ``cmake``. To build your own preset you may start with the preset used in our CI ``presets/docker/CMakePresets.json``.
```

As you already said, I plan to add more documentation for this. So, this comment may be enough in the meantime.